openeo/Pre&PostCovid.ipynb

```diff
@@ -0,0 +1,358 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9a5a4cf7-cf0d-49bf-8229-84d94bafe4dd",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### NO2 emission and COVID Lockdown effects\n",
+    "\n",
+    "The COVID-19 pandemic has had a significant effect on reducing both human and industrial activities, leading to certain positive outcomes such as a decrease in air pollutants, as discussed in this [paper](https://ieeexplore.ieee.org/abstract/document/9617331). \n",
+    "\n",
+    "Therefore, in this notebook, we aim to explore the trend in air pollution, focusing on the NO2 product of the Sentinel 5P collection. We will simply, compare the situation before and after the onset of COVID-19 lockdown in the Delhi region of India.\n",
+    "\n",
+    "Further information on Sentinel-5P can be found the following links:\n",
+    "\n",
+    "* https://documentation.dataspace.copernicus.eu/Data/SentinelMissions/Sentinel5P.html\n",
+    "* https://www.esa.int/Applications/Observing_the_Earth/Copernicus/Sentinel-5P/Sentinel-5P_brings_air_pollution_into_focus\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "3bf6dcb3-dbdc-4ec9-9248-29977999f2bb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import necessary packages\n",
+    "import openeo\n",
+    "from openeo.processes import min, max, mean\n",
+    "import scipy\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "3b27a99b-5fc5-4486-8d40-d799f927174c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Authenticated using refresh token.\n"
+     ]
+    }
+   ],
+   "source": [
+    "connection = openeo.connect(\"openeo.dataspace.copernicus.eu\").authenticate_oidc()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca5d0576-8b2e-4984-b531-5bc1b0add5dd",
+   "metadata": {},
+   "source": [
+    "### 1. Create a datacube for period during COVID lockdowns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "aa36882e-a056-4a9c-835c-e5d4f223c586",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "aoi = {\n",
+    "    \"type\": \"FeatureCollection\",\n",
+    "    \"features\": [\n",
+    "        {\n",
+    "            \"type\": \"Feature\",\n",
+    "            \"properties\": {},\n",
+    "            \"geometry\": {\n",
+    "                \"coordinates\": [\n",
+    "                    [\n",
+    "                        [77.11, 28.69],\n",
+    "                        [77.11, 28.56],\n",
+    "                        [77.29, 28.56],\n",
+    "                        [77.29, 28.69],\n",
+    "                        [77.11, 28.69],\n",
+    "                    ]\n",
+    "                ],\n",
+    "                \"type\": \"Polygon\",\n",
+    "            },\n",
+    "        }\n",
+    "    ],\n",
+    "}"
```

Comment:
You use a feature collection here, but you can just use the inner polygon
that will save on a lot of non-interesting lines of code

Reply:
updated as suggested